bundles/org.eclipse.ui.workbench/eclipseui/org/eclipse/ui/internal/statushandlers/SupportTray.java

```diff
@@ -142,40 +142,39 @@ protected Control createContents(Composite parent) {
 	 */
 	private void createImages() {
 		Display display = Display.getCurrent();
-		int[] shape = new int[] { 3, 3, 5, 3, 7, 5, 8, 5, 10, 3, 12, 3, 12, 5, 10, 7, 10, 8, 12, 10, 12, 12, 10, 12, 8,
-				10, 7, 10, 5, 12, 3, 12, 3, 10, 5, 8, 5, 7, 3, 5 };
 
-		/*
-		 * Use magenta as transparency color since it is used infrequently.
-		 */
+
 		Color border = display.getSystemColor(SWT.COLOR_WIDGET_DARK_SHADOW);
 		Color background = display.getSystemColor(SWT.COLOR_LIST_BACKGROUND);
 		Color backgroundHot = new Color(display, new RGB(252, 160, 160));
-		Color transparent = display.getSystemColor(SWT.COLOR_MAGENTA);
-
-		PaletteData palette = new PaletteData(
-				new RGB[] { transparent.getRGB(), border.getRGB(), background.getRGB(), backgroundHot.getRGB() });
-		ImageData data = new ImageData(16, 16, 8, palette);
-		data.transparentPixel = 0;
-
-		normal = new Image(display, data);
-		normal.setBackground(transparent);
```

Comment:
The setBackground call on the image has been omitted because it has no visible impact; the GC sets the image background to transparent.